Libplanet.Tools/download.mjs

```diff
@@ -49,31 +51,22 @@ export async function download(options = {}) {
     await chmod(dstPath, 0o755);
   };
 
-  const unarchive = (value) => {
-    return new Promise((resolve, reject) => {
-      let retVal;
-
-      if (suffix.toLowerCase().endsWith(".zip")) {
-        retVal = unzipper.Extract({ path: dirPath });
-        retVal.on("close", err => {
-          if (err) return reject(err);
-          return resolve();
-        });
-      } else {
-        const subproc = child_process.spawn("tar", ["xvJ"], {
+  const unarchive = (archivePath) => {
+    if (suffix.toLowerCase().endsWith(".zip")) {
+      return extractZip(archivePath, { dir: dirPath });
```

Comment:
is it okay that return a value instead a promise? (`unarchive` seems to be used with `await`).

Reply:
Yes, `extractZip` is an async function.